`ProtoParserTools.readBytes()` enforces a limit smaller than max bytecode size

### Description

The `0.7.20` version of PBJ runtime is throwing an exception,
```
Caused by: com.hedera.pbj.runtime.ParseException: size 22087 is greater than max 21504
	at com.hedera.pbj.runtime.ProtoParserTools.readBytes(ProtoParserTools.java:274) ~[pbj-runtime-0.7.20.jar:?]
```

But EVM bytecode can be up to 24576 bytes, so this does not work for Services.

### Steps to reproduce

Try to load a state with a bytecode near max size.

### Additional context

_No response_

### Hedera network

other

### Version

`develop`

### Operating system

None

--- a/pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/Field.java
+++ b/pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/Field.java
@@ -14,7 +14,7 @@ import static com.hedera.pbj.compiler.impl.Common.*;
 public interface Field {
 
 	/** The default maximum size of a repeated or length-encoded field (Bytes, String, Message, etc.). */
-	public static final long DEFAULT_MAX_SIZE = 21 * 1024;
+	public static final long DEFAULT_MAX_SIZE = 2 * 1024 * 1024;
 
 	/**
 	 * Is this field a repeated field. Repeated fields are lists of values rather than a single value.

